src/server/hook_func.c

```diff
@@ -5103,25 +5115,29 @@ sync_mom_hookfiles(void *minfo)
 
 		pbs_errno=0;
 		for (j=0; j < minfo_array[i]->mi_num_action; j++) {
+			hook *phook;
+
 			pact = minfo_array[i]->mi_action[j];
 
 			if ((pact == NULL) ||
 				(pact->action == MOM_HOOK_ACTION_NONE))
 				continue;
 
-			if (pact->action & MOM_HOOK_ACTION_DELETE) {
+			phook = find_hook(pact->hookname);
+			if (phook == NULL)
+				continue;
+
+			if (pact->action & MOM_HOOK_ACTION_DELETE)
+				action_expected++;
+
+			if (pact->action & MOM_HOOK_ACTION_SEND_ATTRS)
+				action_expected++;
+
+			if (pact->action & MOM_HOOK_ACTION_SEND_SCRIPT)
+				action_expected++;
+
```

Comment:
Lots of blank lines in this area. I think we can remove some of this whitespace without impacting readability.